### ISSUE TITLE
XCTestCase is not selected

### ISSUE THREAD

**Issue by lmnoel** [uchicago-mobi/MPCS51032-2019-Spring-Forum#35]
I am having an issue where my test cases are not found. I have one test case:

```
    func test_example() {
        XCTAssert(false)
    }
```
and when the tests are run:

```
Test Suite 'Selected tests' started at 2019-05-30 11:52:32.134
Test Suite 'FakeOutTests.xctest' started at 2019-05-30 11:52:32.135
Test Suite 'FakeOutTests.xctest' passed at 2019-05-30 11:52:32.136.
	 Executed 0 tests, with 0 failures (0 unexpected) in 0.000 (0.001) seconds
Test Suite 'Selected tests' passed at 2019-05-30 11:52:32.136.
	 Executed 0 tests, with 0 failures (0 unexpected) in 0.000 (0.003) seconds
```

Latest code is in `development` branch.

**Comment by susanstevens** [comment]
What you have looks right -- no issues are immediately jumping out at me. For some reason, I'm having trouble building your project, though. I cloned your repo, ran `pod install` and then opened the workspace and tried to build. Xcode is giving me this error:

```
error: Unable to load contents of file list: '/Users/.../mpcs-51032-spring-2019-final-project-lmnoel/FakeOut/Pods/Target Support Files/Pods-FakeOut/Pods-FakeOut-frameworks-Debug-input-files.xcfilelist' (in target 'FakeOut')
```

Is there anything special I need to do to get your project to build? (If not, this might just be an Xcode issue, in which case I'll keep investigating...)

**Comment by lmnoel** [comment]
Are you opening the project using `FakeOut.xcodeproj` or `FakeOut.xcworkspace`? I need to use `FakeOut.xcworkspace` for some reason (not sure why). That's the only "special" thing I can think of. I've been fighting Xcode random compiler errors a lot with this project, and I can only assume it's related to the Pods.

**Comment by susanstevens** [comment]
Yep, I was using `FakeOut.xcworkspace`. For some reason I had to do this, https://stackoverflow.com/a/55712251, in order for it to build properly. Seems like an Xcode 10.2 bug.

I also figured out why your tests weren't running. You've got some misplace curly braces. On line 19 of `FakeOutTests.swift`, the closing curly brace for the tear down function is commented out. So everything that comes after it, including the test_example function, is _inside_ the tear down function. Once I fixed the curly braces, test_example ran just fine!
